modules/crafted-init-config.el

```diff
@@ -48,8 +48,11 @@ explicitly."))
 (when (null crafted-emacs-home)
   (setq crafted-emacs-home
         (expand-file-name
-         (project-root
-          (project-current nil (file-name-directory load-file-name))))))
+         (if (executable-find "git")
+             (project-root
+              (project-current nil (file-name-directory load-file-name)))
+           (vc-find-root load-file-name "modules")))))
```

Comment:
Cool!  I wasn't aware of this function, but it looks like this would work nicely.  I'll let @jeffbowman make the final call on merging it.

Reply:
Thanks, I'll look into this.